src/a-runtime-console/kubernetes/store/oauth-config-store.spec.ts

```diff
@@ -118,7 +119,7 @@ describe('OauthConfigStore', () => {
               });
               return new OAuthConfigStore(http, mockUserService, logger, errorHandler, notifications);
             }),
-            deps: [Http, XHRBackend, Logger, ErrorHandler, NotificationsService]
+            deps: [HttpClient, XHRBackend, Logger, ErrorHandler, NotificationsService]
```

Comment:
Yes, i know its more than switching two classes. I've updated quite a lot of tests in previous PRs and this PR also has like 15 specs file updated to `HttpClient` which work properly. The issue with this test is not how to update from `http` to `HttpClient`. The issue is i do not have much knowledge of what this piece of code is doing. Also, the tests are written in a different way by creating a factory out of OAuthStore with different configurations for each `describe` section. I tried to update the tests but couldn't find out the exact URL this service is calling since its relative path `/_config/oauth.json`.

Also, PhantomJS crashes even if a comment out this whole spec file.

Reply:
Can you incrementally comment out (or just use `xdescribe`) specs you've modified to narrow down which spec(s) cause problems?

@jiekang IIRC you looked at the OAuth config store last, do you have any insight on it and its tests here?

Reply:
I can take a look today.

Reply:
On an initial note, PhantomJS crashing even if this spec doesn't exist sounds like there is an issue elsewhere?

Reply:
It seems PhantomJS crash is not happening due to this spec file but some of the UI specs like https://github.com/fabric8-ui/fabric8-ui/blob/400556c00cefbe7d7da1fb52050255e081bcad94/src/a-runtime-console/kubernetes/ui/pipeline/full-history-page/full-history-page.pipeline.spec.ts

Reply:
Some of the tests I found are failing due to a lack of test setup, for example some mocked services don't have methods defined.

Could you attach a log of the PhantomJS crash? So far I've see failing tests but not a full crash of the test runner.